install_config/install/docker_registry.adoc

```diff
@@ -173,20 +173,52 @@ $ oc logs docker-registry-1-da73t
 
 == File Storage
 
-Tag and image metadata is stored in OpenShift, but the registry owns layer and
-signature data. The registry stores data in a volume that is mounted into the
-registry container at *_/registry_*. You can
+Tag and image metadata is stored in OpenShift, but the registry stores layer and
+signature data in a volume that is mounted into the registry container at
+*_/registry_*.
+
+To see a registry's contents, ssh into the pod, then list the contents:
```

Comment:
You're SSH'ing into the node, not the pod. On that note, should probably either be written as "SSH" or marked up with backticks as a command.

But I also think here is where the explanation about this being necessary because it's a privileged container should go. That information is currently commented out below in the IMPORTANT box, but I don't think it should be completely removed.